.github/workflows/create_weekly_sync.yml

```diff
@@ -0,0 +1,58 @@
+name: Create Weekly Sync discussion
+
+# Every Tuesday at 12AM UTC
+on:
+    schedule:
+        - cron: "0 0 * * 2"
```

Comment:
Let me know if a different time works better.

Reply:
Can we do every Wednesday at 12AM? I would like to have it as early as possible but I also don't want to change it if we decide to move the call again. So I think Wednesday for the following week should be enough time.